### DISCUSSION TITLE
Receiving  penidc.so into server: The specified module could not be found.

### DISCUSSION THREAD

**Discussion by AuthNovice** [OpenIDC/mod_auth_openidc#1058] [General]
I've upgraded Apache server on Windows to 2.4.54 and now I can't start up the Web Server receiving the below message.  Does the openidc Module need to be upgraded?
The Apache service named  reported the following error:
>>> penidc.so into server: The specified module could not be found.     .

**Comment by zandbelt** [comment]
support for the Windows build of mod_auth_openidc is only available under a commercial agreement via sales@openidc.com
